tests/cypress/e2e/copyCutPaste.cy.ts

```diff
@@ -0,0 +1,95 @@
+import { GraphqlUtils } from '../utils/graphqlUtils'
```

Comment:
Globally for this test, you can use some function coming from @jahia/cypress to get/add/delete node. There is some function which has been added to do it